queries.sql

```diff
@@ -84,4 +84,32 @@ SELECT animals.* FROM animals LEFT JOIN owners ON animals.owner_id = owners.id L
 SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
 
 -- Who owns the most animals?
-SELECT o.full_name, COUNT(*) AS animal_count FROM owners o JOIN animals a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY animal_count DESC LIMIT 1;
\ No newline at end of file
+SELECT o.full_name, COUNT(*) AS animal_count FROM owners o JOIN animals a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY animal_count DESC LIMIT 1;
+
+-- JOIN TABLE
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.name FROM visits JOIN vets ON visits.vets_id = vets.id  JOIN animals ON visits.animals_id = animals.id WHERE vets.name = 'Vet William Tatcher' ORDER BY date_of_visit DESC LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(date_of_visit) FROM visits JOIN vets ON vets_id = vets.id WHERE vets.name = 'Vet Stephanie Mendez'; 
```

Comment:
- [ ] Here we have to count the no of different animals did Stephanie Mendez see?, & you have calculated the no. of visits , this is wrong, kindly correct it.
```
SELECT DISTINCT animals.name
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'Stephanie Mendez';
```